### DISCUSSION TITLE
Kingroon KP3S profiles

### DISCUSSION THREAD

**Discussion by mechano** [OrcaSlicer/OrcaSlicer#1656] [Ideas]
Good Morning. 

I don't see any profile for Kingroon printers. They make the KP3S a best seller of the small cantilever printers.

Here I attach some good profiles made for Prusa Slicer.

[KP3S Profiles for PrusaSlicer.zip](https://github.com/SoftFever/OrcaSlicer/files/12233921/KP3S.Profiles.for.PrusaSlicer.zip)

**Comment by mechano** [comment]
On Linux version of Orca there're no Kingroon printers. This brand is on
Windows version.

Il giorno ven 8 dic 2023 alle ore 01:25 Carlos Caruncho <
***@***.***> ha scritto:

> There are 2 profiles. Kp3s pro S1 and V2
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SoftFever/OrcaSlicer/discussions/1656#discussioncomment-7794914>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA2ZUDA6TIUOUSES5JDYIJM57AVCNFSM6AAAAAA3AKD5NCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TOOJUHEYTI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Reply by Cftok-Main** [reply]
Dumb question...
For Linux the Orca-Slicer is a *.appimage
How can i put these profile you posted into Orca?
I dont found a possibility to do that.
All Videotutorials are for Windows. And for other printers.

**Reply by mechano** [reply]
There's a way to open the AppImage and do some mods. But I've been told that with next version will be fixes.

**Reply by Cftok-Main** [reply]
I had do that, but without any effect, maybe because they were cura-profiles.

So i tested it with cura appimage and without effect too.

I found a orca-profile but its simply the standard "my klipper printer" settings. Not really a profile for KP3S pro v2.

**Reply by Cftok-Main** [reply]
Ahh i found a profile at kingroons website. The archive contains 10 profiles wirh printer snd filaments.
I imported it in Orca and now it runs.
Perfect, thanks a lot.

**Comment by tadeu2** [comment]
Attach the link here, maybe ill add it to orca defaults

**Reply by Cftok-Main** [reply]
KP3S pro V2 Orca Profiles Printer and Filaments:
https://1drv.ms/f/s!AmGcVrncIH3Zk2lBqc72A8oVQf-U?e=ZEUpYw

KLP1 OrcaSlicer Profile :
https://1drv.ms/f/s!AmGcVrncIH3Zk2otPcnHZrOZPGA5?e=eH3KUu

Found it here:
https://kingroon.com/blogs/3d-print-101/orcaslicer-using-on-kingroon-kp3s-pro-v2-and-klp1-3d-printer

**Comment by tadeu2** [comment]
I loaded two profiles in a old release:

Pro s1

V2

Any problem with those?

**Reply by Cftok-Main** [reply]
You mean there are already profiles in an old release? I dont found Kingroon in the profiles in the version i have. I downloaded it from github and i think its version 1.9.0?
But i use orca only some days now, and i have a printer since some weeks.
All is new for me.
I will look when im at home again.

**Comment by tadeu2** [comment]
There are two default profiles for Kings, both s1 and v2, they are to quick
start.

**Comment by tadeu2** [comment]
If they arent on linux ill include them when i can. Thanks

**Comment by mechano** [comment]
In Windows version of Orca they are present and selectable.

In Linux AppImage they are present in printer files but not selectable in
the user interface.

Il giorno gio 15 feb 2024 alle ore 07:35 Carlos Caruncho <
***@***.***> ha scritto:

> There are two default profiles for Kings, both s1 and v2, they are to
> quick
> start.
>
> El jue, 15 feb 2024, 6:42, Cftok-Main ***@***.***> escribió:
>
> > You mean there are already profiles in an old release? I dont found
> > Kingroon in the profiles in the version i have. I fownloaded it from
> github
> > and i think its version 1.9.0?
> > But i use orca only some days now, and i hsve a printer since some
> weeks.
> > All is new for me.
> > I will look when im at home again.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/SoftFever/OrcaSlicer/discussions/1656#discussioncomment-8475236>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AENP5KRQADYE32SJKBSUBMDYTWN3RAVCNFSM6AAAAAA3AKD5NCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DINZVGIZTM>
>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SoftFever/OrcaSlicer/discussions/1656#discussioncomment-8475523>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA6D45FGXNYAEDR4MH3YTWUCFAVCNFSM6AAAAAA3AKD5NCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DINZVGUZDG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Reply by Cftok-Main** [reply]
Ok, so the appimage must extracted with --extract command and then in folder printerfiles (or any name, i dont know). Then the files can be imported in Orca.

Yesterday i tried to use my Anycubic Kobra 2 pro with orca. Theres a standardprofile for Kobra 2. And at anycubic i found a profile and materials for Kobra 2 plus.

Little problem, orca slices a part veeryy low if i have the Kobra 2 or Kobra 2 Plus selected and generic PLA.
Same part, with orca 7 hours, with anycubic slicer 4 hours.
And no change if i change the speedsetting in orca to higher speeds.

I think the issue is the materials maximum flowrate 12mm/s taht is definded in materialprofile? Is that right?
Because the extruder can much more than 12mm/s. And the material in reality can it too, in anycubicslicer its 18mm/s and above.
And in reality the print dont needs that much time.

**Comment by tadeu2** [comment]
Some profiles could be wrong, you should open an issue and i ll be
corrected by Softfever, i think

**Comment by lrdeoliveira** [comment]
Does anyone already have the basic kps3 configuration for orcaSlicer? I'm using kps3 with Klipper and Mainsal. Am I not getting the cure? can anyone help?

**Reply by Alexdad76** [reply]
https://www.thingiverse.com/thing:6568286/files

**Comment by tadeu2** [comment]
Orca have a basic profile for kp3s pro

**Comment by vgdh** [comment]
I've made a pull request with KP3s profile.
I think it will be available in v2.1 release.
Meanwhile you can use it if you download the latest v2.1 dev version.
